src/javascript/jahia.js

```diff
@@ -1,22 +1,17 @@
 // eslint-disable-next-line no-undef, camelcase
 __webpack_public_path__ = `${window.contextJsParameters.contextPath}/modules/dx-commons-webpack/javascript/apps/`;
 
-// Jahia packages
+// Configuration
+import './apollo/register';
+
+// Global values to expose in Jahia library
+import {startAppShell as startAppShellFunc} from './appShell';
 import * as uiExtenderLib from '@jahia/ui-extender';
 import {default as i18next} from './i18n/i18n';
-import './apollo/register';
-import {load as oldLoader} from './oldLoader';
-import {load as loadAppShell} from './appShell';
 import * as moonstoneLib from '@jahia/moonstone';
 
-export function startAppShell(js, appshellmode, targetId) {
-    if (appshellmode) {
-        return loadAppShell(js, targetId);
-    }
-
-    oldLoader(js);
-}
-
+// Theses value will be exposed in window.jahia
+export const startAppShell = startAppShellFunc;
```

Comment:
why all these const renaming ? can't we simply have export { startAppShell, i18n, uiExtender, moonstone } ?